src/frontend/hooks/useLocation.ts

```diff
@@ -118,3 +142,24 @@ export function getCoords(location: LocationObject): [number, number] {
   const {longitude, latitude} = location.coords;
   return [longitude, latitude];
 }
+
+export function getAccuracy(
```

Comment:
What is this being used for? If only for setting the location on an observation, then I think it might be better to keep this logic along with updating the observation location. It feels fragile to rely on hook updates to decide to update something. If using this in other places, I think it might lead to unexpected behaviour for users. Even if accuracy is worse, and the distance moved less than the accuracy, then a user would expect to see their location move.

Reply:
In mapeo mobile, when making an observation we only updated the location when the accuracy got better. This is just a selector (it doesn't need to be exported). So this implementation allows the user to decide whether or not they want to account for accuracy when updating

Reply:
Will be addressed in [this](https://github.com/digidem/comapeo-mobile/issues/402)